src/Sulu/Bundle/ResourceBundle/Resources/config/doctrine/Condition.orm.xml

```diff
@@ -10,7 +10,7 @@
         <field name="field" type="string" length="255" nullable="false"/>
         <field name="operator" type="string" length="255" nullable="false"/>
         <field name="type" type="integer" nullable="false"/>
-        <field name="value" type="string" length="255" nullable="false"/>
+        <field name="value" type="json_array" nullable="false"/>
```

Comment:
this make the need of upgrading the data maybe its enough to add `"` before and after the value in the database.